### ISSUE TITLE
group chat

### ISSUE THREAD

**Issue by sonn-gamm** [nonlinear-thinkering-group/pask#4]
i think all we need is to put together all the dat folders into one shared node stream??

maybe easier to do if you want to switch over from hyperdrive to hypercore, idk.

i wonder how this would affect the UI, should we add tabs? and how do you create a group if the basic syntax to start a 1:1 chat with someone is `/listen dat-key`.

**Comment by wolkenmachine** [comment]
Yeah, so the issue here is that every user needs to follow back every other user in the group.

One solution could be that once you have two users following each other, they could publish the keys of the other user in the group, so then every new user only needs to "manually" follow one of the users in the group.

Then again, multiwriter dat would basically solve all this, so it depends when we would get that working...

**Comment by sonn-gamm** [comment]
I think then it makes more sense to wait... for dat multiwriter? It seems to be almost there.

And meanwhile we can test other stuff.

**Comment by sonn-gamm** [comment]
i was thinking actually, that having to manually `/follow` the users you want to talk to is fine. what we need maybe, is an option that shows to a user starting following two more users only at a certain point, only the chat that began from that very point in time?

**Comment by wolkenmachine** [comment]
Do you mean that if two people are talking for a while, and they add a third person, that person wouldn't be able to see the conversation the two had previous to that moment?

**Comment by sonn-gamm** [comment]
well yes, which is a bit rude, but it connects to the question of making ad-hoc groups for specific / sensible conversations, and somehow split up a user's chat stream into "places" (eg threads).

this also brings up the question of do we need to set up a private message action?

in a way you can think about it as: each user writes to their dat (chat stream), and then they share specific portions of it depending on the composition of other users they want to talk with?

what are your thoughts?

**Comment by wolkenmachine** [comment]
I like how you talk about "places", though I don't think they are the same as `threads`. I would say a `thread` is a chain of messages. The place is more like `context`, which is different from `topic`.
